RevenueCatUI/Templates/V2/Components/Stack/StackComponentViewModel.swift

```diff
@@ -167,6 +170,15 @@ struct StackComponentStyle {
                                   stackBorder: self.border,
                                   badgeViewModels: badgeViewModels,
                                   uiConfigProvider: uiConfigProvider)
+
+        self.scrollable = overflow.flatMap({ overflow in
```

Comment:
This is just a huge nit, but for me it has been a TIL. I usually use the `map` method for this, and I didn't know the existance of `flatMap` in this context. It turns out that `flatMap` is meant to be used when the closure returns an `Optional`, in order to flatten the 2 optionals. In this case, since the closure returns a nonoptional `Bool`, `map` is more appropriate. Again, feel free to not change this, but I still think this is an interesting caveat 😊 (that I didn't know until now)